static/js/publisher/release/components/releasesTable/channelHeading.js

```diff
@@ -329,6 +339,7 @@ const ReleasesTableChannelHeading = props => {
             promoteToChannel={promoteRevisions}
             channel={channel}
             closeChannel={canBeClosed ? props.closeChannel : null}
+            gaEvent={triggerGAEvents}
```

Comment:
Can we call the prop the same throughout? So this would be `gaEvent` -> `triggerGAEvent`